tests/test_black.py

```diff
@@ -137,6 +137,8 @@ def invokeBlack(
 
 
 class BlackTestCase(BlackBaseTestCase):
+
+    maxDiff = None
```

Comment:
Why do you need this?

Reply:
Forgot to remove this, when running the tests in pycharm, it wasn't showing the diff for the `.diff` tests as the diff between the required diff and actual diff was more than the `maxDiff`. To view the diff, I had to set it to `None`.